pkg/connection/client.go

```diff
@@ -50,18 +50,18 @@ func clientOptions(settings Settings) ([]option.ClientOption, error) {
 // RegistryClient is a client of the Registry API
 type RegistryClient = *gapic.RegistryClient
 
-// NewRegistryClient creates a new GAPIC client using environment variable settings.
+// NewRegistryClient creates a new client using the active Config.
 func NewRegistryClient(ctx context.Context) (RegistryClient, error) {
-	settings, err := ActiveSettings()
+	config, err := ActiveConfig()
```

Comment:
Have we considered removing this "on demand" loading of config values? In most (all?) cases we create one client at the start of the program and pass it along to anywhere it's needed. By loading the config once and setting up the client accordingly, we might be able to eliminate quite a bit of code.

Reply:
I'm unsure where we'd save a lot of code, but I generally like the idea of separating the concerns. We should maybe look at that in another issue, though?